packages/babel-plugin/CHANGELOG.md

```diff
@@ -0,0 +1,135 @@
+# @emotion/babel-plugin
+
+## 11.0.0-next.6
+
+### Major Changes
+
+- [`843bfb11`](https://github.com/emotion-js/emotion/commit/843bfb1153ee0dbe33d005fdd5c5be185daa5c41) [#1600](https://github.com/emotion-js/emotion/pull/1600) Thanks [@Andarist](https://github.com/Andarist)! - Removed `@emotion/css` - it's main purpose was to allow `css` to be a Babel macro, but since babel-plugin-macros allows us to keep imports nowadays this is no longer needed. `@emotion/core/macro` has been added to account for this use case and appropriate changes has been made to `babel-plugin-emotion` to facilitate those changes.
```

Comment:
This is previously generated changelogs (from nodw modified changesets). Should we update it as well? What will happen with those changelogs when exiting pre mode?

Reply:
I think we should probably leave it. Changesets will put all of the changesets in the changelog when exiting pre mode